charts/prometheus-smartctl-exporter/templates/daemonset.yaml

```diff
@@ -35,12 +35,24 @@ spec:
         {{- include "prometheus-smartctl-exporter.selectorLabels" $global | nindent 8 }}
         idx: i{{ $idx }}
     spec:
+      {{- if $global.Values.image.pullSecrets }}
+      imagePullSecrets:
```

Comment:
Please, add the new field `image.pullSecrets` with its default value in `values.yaml`.